Fix vct in metadata (#356)
oauth-wg/oauth-sd-jwt-vc: draft-ietf-oauth-sd-jwt-vc.md

The Type Metadata document MUST be a JSON object. The following properties are
defined:
* `vct`
  * REQUIRED. The verifiable credential type described by this type metadata document.

>     The value MUST be identical to the `vct` claim used in the SD-JWT VC payload.

Reviewer: Is this "The value MUST be identical to the `vct` claim used in the SD-JWT VC payload" problematic with metadata inheritance?

## Retrieving Type Metadata {#retrieving-type-metadata}

A consumer retrieving Type Metadata MUST ensure that the `vct` value in the
SD-JWT VC payload is identical to the `vct` value in the reference to the Type
Metadata (either in the SD-JWT VC itself or in an `extends` property in the Type
Metadata document).

> The following methods can be used to retrieve Type Metadata:

Author: ```suggestion
The following sections define methods to retrieve Type Metadata.
```
